service/frontend/dcRedirectionHandler.go

```diff
@@ -1211,6 +1211,62 @@ func (handler *DCRedirectionHandlerImpl) GetSystemInfo(
 	return handler.frontendHandler.GetSystemInfo(ctx, request)
 }
 
+// CreateSchedule API call
```

Comment:
Schedule APIs should follow the regular APIs to be forwarded if current cluster is not active.